roles/foreman_installer/tasks/install.yml

```diff
@@ -3,7 +3,7 @@
   shell: >
       {{ foreman_installer_command }} {{ (foreman_installer_verbose == True) | ternary("-v", "") }}
       --scenario "{{ foreman_installer_scenario }}"
-      {{ foreman_installer_options }}
+      {{ foreman_installer_options | join(" ") }}
```

Comment:
Doesn't this break when it's a string?

Reply:
Yes it does. It separates each character in the string with a `" "`

Reply:
I'd vote to put this in a separate PR (I like the change to using an array of stings vs. one big ole string). We'd need to change any places in forklift that are using the string of options.

Reply:
:/  From grepping, this new playbook is literally the only thing using this option and if possible, I'd prefer not to write the luna options as a giant ugly string only to then to then re-write them back to an array.

Reply:
That's because this option was always intended as a *user* option. You'll find this in `boxes.d/*` which is not in git. You're right that an array is better, but just changing it will break users configs.

Reply:
I will make this change, but not because I agree with it. This repository is unversioned, this is relatively unused parameter and this just adds technical debt cruft rather than moving towards a better solution for users that will be easy to adapt to. So before I make this change, the question is when can we remove this backwards compatibility layer? 1 month? 2 months?

Reply:
I'm fine with making the change and breaking current box configs if you make sure to notify the forklift users (i.e. community post?).

Didn't you face the same dilemma when you moved the vagrant directory? Isn't [this](https://github.com/theforeman/forklift/blob/master/Vagrantfile#L9) tech debt? And when should we remove it? ;)

Reply:
@ehelms I'd also be fine with a hard error instructing users to move to a list rather than a string. Forcing users to upgrade generally makes the time to remove it shorter.

Reply:
I like that idea, @ekohl. Maybe we should also make an issue on forklift to list the tech debt we need to eventually remove.